react-native-upload.podspec

```diff
@@ -7,7 +7,7 @@ require "json"
    s.name = package[:name]
```

Comment:
The new package name won't work as the podspec package name.

I ran into this issue with react-native-payments.
See this commit. I recommend doing something similar.
https://github.com/appfolio/react-native-payments/commit/bd8b296b7b29cc2ce341bdf394d8ee81f6890c10